_posts/2024-05-10-interactive-bioinformatics.md

```diff
@@ -0,0 +1,70 @@
+# Interactive bioinformatics
+
+This blog was suggested by Helena and inspired a question on gamification of bioinformatics. So buckle up as we look at a wide range of very cool educational tools and projects.
+
+## Educational
+
+- [TeachEnG - Illinois Unversity](https://teacheng.illinois.edu/)
+A collection of interactive web-based games for learning sequence alignment, dot plots and phylogenetic trees. 
+
+- [Full spectrum Bioinformatics](https://github.com/zaneveld/full_spectrum_bioinformatics) - Little Brother is Missing
+Learn how to navigate folders and files through the CLI by playing a spooky game.
+
+- [Bashcrawl](https://gitlab.com/slackermedia/bashcrawl)
+Learn CLI by playing a dungeon crawler.
+
+- [PonyLinux](https://github.com/NCGAS/PonyLinux)
+Another CLI game but with Ponys.
+
+- https://sandbox.bio/
+Perform actual bioinformatics tasks such as alignment and variant calling all in the browser. Several tutorials on alignment, variant calling, CLI and even jq. 
+All through the magic of wasm (biowasm). The exploration sandboxes are worth checking out if you want to grok how some algorithms work. 
+Worthy of it's own blog post with the number of tutorials.
+
+- The Great Clade Race - Zombie Island Edition  https://serc.carleton.edu/teachearth/activities/238347.html
+Learn cladistics theory through a fun game zombie game. This is particularly interesting as it's more DnD-like.
```

Comment:
```suggestion

Resource | About
--- | ---
[Full spectrum Bioinformatics](https://github.com/zaneveld/full_spectrum_bioinformatics) | Little Brother is Missing: Learn how to navigate folders and files through the CLI by playing a spooky game.
[Bashcrawl](https://gitlab.com/slackermedia/bashcrawl) | Learn CLI by playing a dungeon crawler, fight monsters, and pick up spells to help you explore along the way
[PonyLinux](https://github.com/NCGAS/PonyLinux) | Another CLI game but with Ponys! :carousel_horse: 

Leaving the terminal and moving to bioinformatics itself, there are other interactive resources that can make learning a bit more fun such as:

- [TeachEnG - Illinois Unversity](https://teacheng.illinois.edu/), which provides a collection of interactive web-based games for learning sequence alignment, dot plots and phylogenetic trees.
- [The Great Clade Race - Zombie Island Edition](https://serc.carleton.edu/teachearth/activities/238347.html),
learn cladistics theory through a fun game zombie game. This is particularly interesting as it's more DnD-like.

 ## Bioinformatics in the Browser
  
[sandbox.bio](https://sandbox.bio/) is a very cool new resource which runs bioinformatics tools directly in the browser! You can perform actual bioinformatics tasks such as alignment and variant calling all in the browser. Several tutorials on alignment, variant calling, CLI and even jq. All through the magic of wasm (biowasm). The exploration sandboxes are worth checking out if you want to grok how some algorithms work. Note that it will work best on laptops and desktops that have the memory and cpu to allocate to these tools. While WASM is constantly improving there is overhead to trying to run everything in the browser.
```